src/transport/PointToPointBroker.cpp

```diff
@@ -566,10 +672,74 @@ std::vector<uint8_t> PointToPointBroker::recvMessage(int groupId,
 
     Message message = recvEndpoints[label]->recv();
```

Comment:
This message will be the body message, but the sequence number is in the header. We can add a bit in the transport layer to set it on the body.